src/ServiceControlInstaller.Engine/Configuration/ServiceControl/SettingsList.cs

```diff
@@ -65,5 +65,11 @@ public static class ServiceControlSettings
             Name = "ServiceControl/MaintenanceMode",
             SupportedFrom = new Version(1, 18, 1)
         };
+
+        public static SettingInfo RemoteInstances = new SettingInfo
+        {
+            Name = "ServiceControl/RemoteInstances",
+            SupportedFrom = new Version(1, 47, 0)
```

Comment:
Shouldn't this be 2.0?

Reply:
Not according to https://github.com/Particular/ServiceControl/releases/tag/1.47.0